prod/ops.yml

```
@@ -27,3 +27,11 @@
       inputs:
         procstat:
           exe: concourse
+
+- type: replace
+  path: /instance_groups/name=db/jobs/-
```

this is gonna add a new job entry - i think you just want this?:

```suggestion
  path: /instance_groups/name=db/jobs/name=postgres/properties/databases/additional_config?/max_parallel_workers_per_gather
```

(...and then `value: 0`)